app/quadraticlands/views.py

```diff
@@ -295,3 +298,112 @@ def error(request, code):
     if return_as_json:
         return JsonResponse(context, status=code)
     return TemplateResponse(request, f'quadraticlands/error.html', context, status=code)
+
+
+def mission_diplomacy(request):
+    return mission_diplomacy_helper(request)
+
+def mission_diplomacy_helper(request, invited_to_game=None):
+    games = []
+    max_games = 4
+
+    # check for what games the user is in
+    if request.user.is_authenticated:
```

Comment:
remove and use login_required decorator

Reply:
nah bc we want ppl to be able to view the index ( and git QD-pilled) before logging in